astropy/table/table_helpers.py

```diff
@@ -96,9 +96,9 @@ def simple_table(size=3, cols=None, kinds='ifS', masked=False):
     letters = np.array([c for c in string.ascii_letters])
     for jj, kind in zip(range(cols), cycle(kinds)):
         if kind == 'i':
-            data = np.arange(1, size + 1, dtype=int) + jj
+            data = np.arange(1, size + 1, dtype=np.int64) + jj
```

Comment:
More out of curiosity than anything else: Is this really the underlying problem?  Or is it just the `dtype=kind` line? On my machine at least:
```
In [2]: np.arange(10, dtype=int).dtype
Out[2]: dtype('int64')
In [3]: Column(np.arange(10, dtype=int), dtype='i').dtype
Out[3]: dtype('int32')
```
Or is this different on a 32-bit machine? Anyway, explicit is better than implicit, so no reason to change!